Adding lynx to Dockerfile

fdns opened the pull request:
lynx is required for emails, and is not installed in the Dockerfile currently

https://github.com/edx/edx-platform/blob/b489a4ecb12252099b1d249e588158ff038187a1/openedx/core/lib/html_to_text.py#L16

edx-webhook commented:
Thanks for the pull request, @fdns! I've created [OSPR-5058](https://openedx.atlassian.net/browse/OSPR-5058) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://open.edx.org/wp-content/uploads/2019/01/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

<!-- comment:no_cla -->


<!-- comment:external_pr -->
<!-- data: eyJkcmFmdCI6IGZhbHNlfQ== -->

natabene commented:
@fdns Thank you for the contribution! When you have a chance, could you please send in a signed Contributor Agreement mentioned in the earlier comment? You only need to do this once and we can then review all your future code contributions. Thanks

natabene commented:
@fdns Thank you for joining the Open edX community! We have added your user to our records and can now review your code going forward.

natabene commented:
jenkins run all

natabene commented:
@fdns Can you please look into the failing python test?

bradenmacdonald commented:
@fdns Thanks for this! Would you mind rebasing (to get the tests passing) and also adding a comment in the Dockerfile with a link to `https://github.com/edx/edx-platform/blob/b489a4ecb122/openedx/core/lib/html_to_text.py#L16` so that we know in the future why lynx was added and that it can be removed if that code is updated to "something more pythonic" ?

@natabene OK if I do the review here?

natabene commented:
@bradenmacdonald Done.

fdns commented:
Just rebased and added the comment.

We can update the list at https://github.com/edx/edx-platform/blob/master/requirements/system/ubuntu/apt-packages.txt and reference it for the installation later (lynx-cur is not longer a valid package for example).

bradenmacdonald commented:
Thanks!
Just trying to fix the reporting of the tests here, not sure what's up with some of them.
jenkins run py38 js
jenkins run py38 a11y
jenkins run js
jenkins run a11y

bradenmacdonald reviewed (approved):
:+1: Thanks for this contribution!

- [x] I tested this: Ran `make docker_build`, confirmed that the `RUN apt-get install` step works. It installs `lynx-common (2.9.0dev.5-1)` and `lynx (2.9.0dev.5-1)`
- [x] I read through the code
- [x] I checked for accessibility issues: not applicable
- [x] Includes documentation: Yes, now in a comment.

bradenmacdonald commented:
Not sure if I'll get the test issues sorted out today; may merge this on Monday.

edx-webhook commented:
@fdns 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-platform&entry.867055334=https://github.com/edx/edx-platform/pull/25343&entry.1484655318=https://github.com/fdns&entry.752974735=2020-10-14+00:07&entry.1917517419=2020-11-17+00:09&entry.2133058324=Yes) so we can improve your experience in the future.
